debian/control

```diff
@@ -94,9 +103,16 @@ Description: real-time visual space simulation (GLUT frontend)
  .
  This package contains a minimal frontend for Celestia.
 
+Package: libcelestia
```

Comment:
libcelestia is installed like a system library?
so under /usr/lib/ or /usr/lib64/ ?
does debian/ubuntu require installation instructions such as debian/libcelestia.install ?
the names would be something like
`libcelestia.so.1 -> libcelestia.so.1.7.0`

Reply:
> libcelestia is installed like a system library?
> so under /usr/lib/ or /usr/lib64/ ?

yes, it sounds like reasonable location to me.

> does debian/ubuntu require installation instructions such as debian/libcelestia.install ?

I forgot to add it to git. fixed.

> libcelestia.so.1 -> libcelestia.so.1.7.0

SONAME should be updated on breaking API changes, so we'll have  `libcelestia.so.1.7 -> libcelestia.so.1.7.0`